incubator/nodejs-loopback/templates/scaffold/src/__tests__/README.md

```diff
@@ -0,0 +1,3 @@
+# Tests
```

Comment:
we could possibly remove this readme?

Reply:
We generate these files to guide developers to understand purposes of such folders. I would like to keep them to be consistent with `@loopback/cli`.